src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/services/ReportedAdjudicationServiceTest.kt

```diff
@@ -427,6 +519,141 @@ class ReportedAdjudicationServiceTest {
       }
     }
 
+    // TODO - This is too complicated - may as well revert to the existing way
```

Comment:
This is too complex but I haven't figured out a way to improve - maybe separate the params in to 2 or 3 tests and have common methods to set the data up and check it?